aten/src/ATen/core/Tensor.h

```diff
@@ -68,7 +68,7 @@ class CAFFE2_API Tensor {
   }
 
   bool defined() const {
-    return impl_;
+    return impl_ && impl_->defined();
```

Comment:
as mentioned on the other page, I don't understand how we get in this situation.